### ISSUE TITLE
Accessibility: empty avatar link on people index

### ISSUE THREAD

**Issue by crowesn** [uclibs/scholar_uc#1683]
### Descriptive summary

Users with no avatar have a link with no text in the avatar column on people index, resulting in accessibility violation.

Best to suppress the link.

```
<tbody>
<tr>
   <td><a href="/users/scanner@example-dot-com?locale=en">
   </a>              </td>
   <td><a href="/users/scanner@example-dot-com?locale=en">Scanner Account</a></td>
   <td><a href="/users/scanner@example-dot-com?locale=en">scanner@example.com</a></td>
   <td>Unknown</td>
   <td>1</td>
   </tr>
</tbody>

```

https://wsuofc.dequecloud.com/worldspace/issue/edit/3747098
